packages/cli-repl/src/cli-repl.ts

```diff
@@ -454,6 +458,17 @@ class CliRepl {
     );
   }
 
+  /**
+   * Is the appName missing from the connection string?
+   *
+   * @param {ConnectionString} cs - The existing connection string.
+   *
+   * @returns {boolean} If the appName is missing.
+   */
+  isAppNameMissingURI(cs: ConnectionString): boolean {
+    return !cs.searchParams.get('appName');
```

Comment:
I think this can be inlined into the source – `!cs.searchParams.get('appName')` essentialls means “is the appName missing from the URI” :)